feat(styles): update stepper (#1646)
swisspost/design-system: .changeset/rare-kids-drum.md

Reviewer: It was me who asked to separate the changeset because it seems strange to me to have "clarified how to use the stepper" in the styles changelog... But it is true that the second changelog does not specify which component is affected by the changes.

Author: Is there a convention for the context, or is it fine like this?

---
'@swisspost/design-system-styles': patch
---

Stepper:

- Normalize font styles and weights.

> - Position the step label under the the badge on small viewport.

Reviewer: ```suggestion
- Position the step label under the badge on small viewport.
```

> - Center the step label when this text is multiline.

Reviewer: ```suggestion
- Center the step label when its is multiline.
```
